Makefile

```diff
@@ -291,6 +291,15 @@ test-cephs3: tester
 	@$(call remove_docker_network)
 .PHONY: test-cephs3
 
+test-nsfs-cephs3: tester
+	@echo "\033[1;34mRunning tests with Postgres.\033[0m"
+	@$(call create_docker_network)
```

Comment:
I don't think we are actually starting up postgres here. Also, do we need a custom docker network here 🤔 ?

Reply:
Looks this needs to be removed. This was left because earlier it was planned different and implementation changed after standalone nsfs implementation

Reply:
we need docker network

`docker: Error response from daemon: network noobaa-net not found.`

Reply:
Yeah that's because in the next line the network is being passed under --network flag. That flag itself might not be needed as well as we aren't attaching any other container to the network.

Reply:
Thats right.